django_dramatiq/management/commands/rundramatiq.py

```diff
@@ -67,8 +74,13 @@ def handle(self, use_watcher, use_polling_watcher, use_gevent, path, processes,
             executable_name,
             "--path", *path,
             "--processes", str(processes),
-            "--threads", str(threads),
+            "--threads", str(threads)
+        ]
+
+        if queues:
```

Comment:
This flag should be appended at the very end of the argument list, otherwise it's ambiguous in conjunction with the `MODULE_NAME` positional arg:

```
dramatiq --threads 8 --proceses 8 --queues a b c d module_a module_b
```

has a different meaning from

```
dramatiq --threads 8 --proceses 8 module_a module_b --queues a b c d
```